packages/cojson/src/CoValuesStore.ts

```diff
@@ -0,0 +1,38 @@
+import { CoValueCore } from "./coValueCore.js";
+import { CoValueState } from "./coValueState.js";
+import { RawCoID } from "./ids.js";
+
+export class CoValuesStore {
+  coValues = new Map<RawCoID, CoValueState>();
+
+  get(id: RawCoID) {
+    let entry = this.coValues.get(id);
+
+    if (!entry) {
+      entry = CoValueState.Unknown(id);
+      this.coValues.set(id, entry);
+    }
+
+    return entry;
+  }
+
+  setAsAvailable(id: RawCoID, coValue: CoValueCore) {
+    const entry = this.get(id);
+    entry.dispatch({
+      type: "available",
+      coValue,
+    });
+  }
```

Comment:
Introduced this store to enforce two things in the CoValues state management:
- That reads always return an entry defaulting to an Unknown state
- Once set an entry is always the same. We never override o delete entries (we will delete entries once we implement the garbage collection system)